patient-portal-ui/src/datajs/DeafutlAllData.tsx

```diff
@@ -125,9 +126,9 @@ export const DeafutlAllData = {
     return data;
   },
   getEventTypes: async function () {
-    let response = await fetch(url_0 + 'public/eventtypes');
-    const data = await response.json();
-    return data;
+    // let response = await fetch(url_0 + 'public/eventtypes');
+    // const data = await response.json();
+    // return data;
```

Comment:
we can remove commented lines if we don't need them, we can always retrieve from commits history in case